activation/atxdb.go

```diff
@@ -692,3 +734,71 @@ func (db *DB) ValidateSignedAtx(pubKey signing.PublicKey, signedAtx *types.Activ
 	}
 	return nil
 }
+
+func (db *DB) HandleGossipAtx(data service.GossipMessage, syncer service.Syncer) {
```

Comment:
do you feel like this really belongs to db ?

Reply:
It definitely belongs in activation package, don't know where exactly but I'm open to suggestions

Reply:
I agree about package but seeing it in `DB` feel a little big weird. even if the solution is just creating a new stupid component that will rely on an interface from "DB". decoupling..

Reply:
DB is now inherently needed in this function since it is the one doing the handling of received ATX, if I reverse the dependencies and have DB as input it will require the caller to know which db to handle. 
The thing is that currently db also handles syntactic validation and also applies logic, so it's not simply a CRUD storage
Maybe we sholud consider a different name for it but I can't think of one... 
AtxHandler? @y0sher

Reply:
I see, I believe renaming ore reversing the dependencies like you said will result in a more understandable design. but we should atleast have some docs about this module and what are its responsibilities.